integration-tests/tests/billing-projects.js

```diff
@@ -131,17 +174,116 @@ const testBillingSpendReportFn = withUserToken(async ({ page, testUrl, token })
   await billingPage.assertChartValue(3, 'Third Most Expensive Workspace', 'Storage', '$0.00')
 
   // Change the returned mock cost to mimic different date ranges.
-  await setAjaxMockValues(page, billingProjectName, '1110.17', 20)
+  await setAjaxMockValues(page, ownedBillingProjectName, notOwnedBillingProjectName, '1110.17', 20)
   await billingPage.setSpendReportDays(90)
   await billingPage.assertText('Total spend$1,110.17')
   // Check that title updated to reflect truncation.
   await billingPage.assertText('Top 10 Spending Workspaces')
   await billingPage.assertChartValue(10, 'Extra Inexpensive Workspace', 'Compute', '$0.01')
+
+  // Select a billing project that is not owned by the user
+  await billingPage.visit()
+  await billingPage.selectProject(notOwnedBillingProjectName)
+
+  //Check that the Spend report tab is not visible on this page
+  await assertTextNotFound(billingPage, 'Spend report')
 })
 
 const testBillingSpendReport = {
   name: 'billing-spend-report',
   fn: testBillingSpendReportFn
 }
 
-module.exports = { testBillingSpendReport }
+const testBillingWorkspacesFn = withUserToken(async ({ page, testUrl, token }) => {
+  // Sign in. This portion of the test is not mocked.
+  await page.goto(testUrl)
+  await signIntoTerra(page, token)
+  await dismissNotifications(page)
+
+  // Interact with the Billing Page via mocked AJAX responses.
+  const ownedBillingProjectName = 'OwnedBillingProject'
+  const notOwnedBillingProjectName = 'NotOwnedBillingProject'
+  await setAjaxMockValues(page, ownedBillingProjectName, notOwnedBillingProjectName, '0')
+
+  const billingPage = billingProjectsPage(page, testUrl)
+
+  // Select a billing project that is owned by the user
+  await billingPage.visit()
```

Comment:
Nit: to remove the duplication put this code in a block that you loop over for the 2 billing project names.